src/Lykke.Common.Validation/LykkeValidation.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Lykke.Common.Validation.Configuration;
+
+namespace Lykke.Common.Validation
+{
+    /// <summary>
+    /// Provides static methods to operate with LykkeValidation configuration
+    /// </summary>
+    public static class LykkeValidation
+    {
+        private static bool _isInitialized;
+
+        /// <summary>Initialize static configuration instance</summary>
+        /// <param name="configAction"></param>
+        public static void Initialize(Action<IValidationConfigurationExpression> configAction)
+        {
+            if (_isInitialized)
```

Comment:
Why don't we check Configuration  for null as a flag?

Reply:
fixed